plugins/lookup/conjur_variable.py

```diff
@@ -86,6 +90,9 @@
 from base64 import b64encode
 from netrc import netrc
 from os import environ
+import stat
+from random import choice
+import string
```

Comment:
Imports need to be sorted alphabetically

Reply:
I sorted the one I added, others don't seem to be though. 
I'll keep it out of this PR.